week4_assignment/login-signup-project/src/pages/Mypage.jsx

```diff
@@ -0,0 +1,234 @@
+import React, { useState, useEffect } from 'react';
+import { useParams, useNavigate } from 'react-router-dom';
+import styled from 'styled-components';
+
+function MyPage() {
+  const [userInfo, setUserInfo] = useState({ id: '', nickname: '', phone: '' });
+  const [showPasswordForm, setShowPasswordForm] = useState(false);
+  const [passwords, setPasswords] = useState({
+    previousPassword: '',
+    newPassword: '',
+    confirmPassword: '',
+  });
+  const { memberId } = useParams();
+  const navigate = useNavigate();
+
+  useEffect(() => {
+    const fetchData = async () => {
+      try {
+        const response = await fetch(`http://34.64.233.12:8080/member/info`, {
+          method: 'GET',
+          headers: {
+            memberId: memberId,
+          },
+        });
+        const data = await response.json();
+        if (response.ok) {
+          setUserInfo({
+            id: data.data.authenticationId,
+            nickname: data.data.nickname,
+            phone: data.data.phone,
```

Comment:
요런거 객체 구조분해할당 활용하면 예쁘게 만져줄 수 있어요 

```suggestion
        const { data : { authenticationId : id, nickname, phone } }  = await response.json();  // 일케 쪼개주면 
        if (response.ok) {
          setUserInfo({
            id,
            nickname,
            phone,
```
일케 해줄 수 있어요이 ~